lib/xandra/cluster.ex

```diff
@@ -186,6 +188,10 @@ defmodule Xandra.Cluster do
     {load_balancing, options} = Keyword.pop(options, :load_balancing, @default_load_balancing)
     {nodes, options} = Keyword.pop(options, :nodes)
     {autodiscovery?, options} = Keyword.pop(options, :autodiscovery)
+
+    {autodiscover_other_datacenters, options} =
+      Keyword.pop(options, :autodiscover_other_datacenters)
```

Comment:
Maybe we can rename the option to be clear it's a boolean:

```suggestion
    {autodiscover_other_datacenters?, options} =
      Keyword.pop(options, :autodiscover_other_datacenters)
```